src/symbols_linux.cpp

```diff
@@ -394,17 +401,19 @@ void ElfParser::parseDynamicSection() {
                     _cc->addImport((void**)(base + r->r_offset), strtab + sym->st_name);
                 }
             }
-        } else if (rel != NULL && relsz != 0) {
+        }
+
+        if (rel != NULL && relsz != 0) {
```

Comment:
Can the same function occur in multiple relocations? I believe it can. We don't necessarily need to handle such cases right now, but if this happens, `.rela.plt` should take precedence.